src/app/pages/jails/jail-wizard/jail-wizard.component.ts

```diff
@@ -73,11 +77,7 @@ export class JailWizardComponent {
           type: 'input',
           name: 'ip4_addr',
           placeholder: T('IPv4 Address'),
-          tooltip: T('Type the IPv4 address for VNET and shared IP jails.\
- Single interface format: <b>interface|ip-address/netmask</b>. Multiple\
- interface format:\
- <b>interface|ip-address/netmask,interface|ip-address/netmask</b>.\
- Example: <b>vnet0|192.168.0.10/24</b>'),
+          tooltip: T('The IPv4 address for the jail to use.'),
```

Comment:
What about "Enter an IPv4 address for the jail." ? Seems to flow a little better.